script/Ciel/Day 1/01_01_CIEL01_1_3.txt

```diff
@@ -86,7 +86,7 @@ Even if I were to somehow obtain the secret of speed itself, there's no way I co
 [sha:fb71879a715b57b70f6d0a1857fe58d551f7f59b]{
 -- Page 18, Offset 539.
 -- 美しいものに惹かれる心理をどう言葉にしろと？
-How can I put into words the psychology of being attracted to something beautiful?
+How can I put into words the mental state of being attracted to something beautiful?
```

Comment:
revert

Reply:
Why

Reply:
because "the psychology of" is a set expression